src/utils/teardown.js

```diff
@@ -1,9 +0,0 @@
-const { stopSourceMapWorker } = require("devtools-source-map");
-const { stopPrettyPrintWorker } = require("../utils/pretty-print");
-const { stopParserWorker } = require("../utils/parser");
-
-export function teardownWorkers() {
-  stopSourceMapWorker();
-  stopPrettyPrintWorker();
-  stopParserWorker();
-}
```

Comment:
I thought it was helpful to see both bootstrap and teardown in the same file. For example with this change, we need to use the same Firefox conditional for the source map worker. With it all in one file, it's easy to see they work the same way.